app/tools/android/make_apk.py

```diff
@@ -142,12 +140,36 @@ def FindExtensionJars(root_path):
         extension_jars.append(extension_jar)
   return extension_jars
 
+# try to parse out suitable app_versionCodeBase based on app_version
+# for example, app_version "3.1.0" will generate app_versionCodeBase "310"
```

Comment:
yes we considered again and think we can parse out versionCodeBase  only for versionName with format xx.xx.xxx, otherwise will notify failure and suggest user to set versionCode or versionCodeBase manually.
The supported versionName regular expression is \d{1,2}\.+\d{1,2}\.+\d{1,3}

for example, 3.2.0 will be 0302000, 3.1.11 will be 0301011, 12.5.178 will be 1205178.